src/redist/minimumMSBuildVersion

```diff
@@ -0,0 +1 @@
+15.3.246
```

Comment:
Is that guarantee to work? I used this one because it is the one inserted into VS. So I know it was a good one.